identity-core/Cargo.toml

```
@@ -15,6 +15,7 @@ base64 = { version = "0.13", default-features = false, features = ["std"] }
 bs58 = { version = "0.4", default-features = false, features = ["std"] }
 hex = { version = "0.4", default-features = false }
 identity-diff = { version = "=0.4.0", path = "../identity-diff", default-features = false }
+js-sys = { version = "0.3.55", optional = true }
```

Even if a crate doesn't have features it just helps to make it easier to check that we're not pulling stuff in unnecessarily.

```suggestion
js-sys = { version = "0.3.55", default-features = false, optional = true }
```